docs/ibm-config.md

```diff
@@ -57,7 +57,9 @@ Specify the following values in the example files:
 
   * Replace `<YOUR_BUCKET>`, `<YOUR_REGION>` and `<YOUR_URL_ACCESS_POINT>`. See the [BackupStorageLocation definition][6] for details.
 
+* (Optional, use only to specify multiple volume snapshot locations) In `examples/aws/10-deployment.yaml` (or `examples/aws/10-deployment-kube2iam.yaml`, as appropriate):
```

Comment:
```suggestion
* (Optional, use only to specify multiple volume snapshot locations) In `examples/ibm/10-deployment.yaml`:
```

I'm not 100% sure this is relevant on IBM - their object storage is supported via the S3 API, but their own block storage isn't, unless the cluster has something like Ceph or Portworx.